website/client/components/chat/autoComplete.vue

```diff
@@ -3,8 +3,8 @@
   .autocomplete-results.d-flex.align-items-center(
     v-for='result in searchResults',
     @click='select(result)',
-    @mouseenter='result.hover = true',
-    @mouseleave='result.hover = false',
+    @mouseenter='resetSelection(); result.hover = true',
```

Comment:
could `result.hover = true` be moved inside `resetSelection `?

Reply:
No. `resetSelection()` sets `hover` to `false` for all the results, so it wouldn't make sense.

Reply:
I suppose it's possible to pass result into the function as an optional parameter, and then we would be able to set it inside the function. Would you prefer that?

Reply:
yeah it would be better, we try to keep the js into templates to a minimum, thanks!

Reply:
@paglias I updated the code. Please take another look